### ISSUE TITLE
Downloads for macOS and Windows are not clear for regular users

### ISSUE THREAD

**Issue by jgallias** [keepassxreboot/keepassxreboot.github.io#48]
The terms "Binary bundle" on the macOS Download page and "Executable installer" on the Windows Download page are too complex for the average user to understand.
Perhaps they could be simplified into "Download" or "Download and Install"?
The most common default options should be larger and above everything else to minimize confusion and reduce error.

Example comparison with 1Password download page:
https://github.com/brianlovin/security-checklist/issues/114#issuecomment-459869157

**Comment by droidmonkey** [comment]
We are going to be doing a minor website overhaul as part of our accessibility push. This will be part of that overhaul. Thank you!
